tests/unit/protocol.tcl

```diff
@@ -60,12 +60,13 @@ start_server {tags {"protocol network"}} {
         assert_error "*wrong*arguments*ping*" {r ping x y z}
     }
 
-    test "Unbalanced number of quotes" {
+    test "Mixing quoted and unquoted strings" {
```

Comment:
We could keep a test that give the error for unbalanced quotes, like `"get 'foo"`, just to cover that error.

Reply:
Added some tests back that should cover unbalanced single and double quotes as well as escaped single quotes in the unit tests.